### PR TITLE
Add Server Medianet To Schema

### PR DESCRIPTION
**Pull request by cristiandiazmrf**

### CODE REVIEW

File: json-lint/schemas/inventory/adservers/medianet.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            "type": "string"
+        },
+        "cid": {
+            "type": "string"
+        },
+        "crid": {
+            "type": "string"
+        },
+        "misc": {
+            "type":[ "object"
```
Comment (ricardoribasmrf) [inline json-lint/schemas/inventory/adservers/medianet.json]: Should be an object no?

File: json-lint/schemas/inventory/adservers/medianet.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                "cid": "$2"
+            }
+        }
+    ],
+    "properties": {
+        "type": {
+            "const": "medianet"
+        },
+        "tagtype": {
+            "type": "string"
```
Comment (ricardoribasmrf) [inline json-lint/schemas/inventory/adservers/medianet.json]: Use enum to use one of the `data-tagtype="cm"` OR `data-tagtype="headerbidder"`

Comment (ricardoribasmrf) [reply json-lint/schemas/inventory/adservers/medianet.json]: > data-tagtype - This parameter represents the product the publisher is using; It should be cm for our Contextual Monetization solution

File: json-lint/test/resources/schema/inventory/valid/medianet.test.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            "type": "medianet",
+            "tagtype": "cm",
+            "cid": "8CU3C25G1",
+            "crid": "790473691",
+            "width": 600,
+            "height": 250
+        },
+        "medianet1": {
+            "type": "medianet",
+            "tagtype": "cm",
```
Comment (ricardoribasmrf) [inline json-lint/test/resources/schema/inventory/valid/medianet.test.json]: Add an example of header bidder

**Comment by joaquimsantmrf** [comment]
retest this please

**Pull request merged**